gapic/src/main/com/google/gapid/views/SettingsDialog.java

```diff
@@ -156,6 +162,15 @@ protected void configureDialog(FileDialog dialog) {
       };
       adbPath.addBoxListener(SWT.Modify, adbListener);
       adbListener.handleEvent(null);
+
+      allowUpdateChecks.addSelectionListener(new SelectionAdapter()
+        {
```

Comment:
braces don't get their own lines in the Google Java Style guide.

Reply:
thanks, done